src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.WorkerThread.cs

```diff
@@ -189,7 +189,14 @@ internal static bool ShouldStopProcessingWorkNow()
                 ThreadCounts counts = ThreadCounts.VolatileReadCounts(ref ThreadPoolInstance._separated.counts);
                 while (true)
                 {
-                    if (counts.numExistingThreads <= counts.numThreadsGoal)
+                    /// When there are more threads processing work than the thread count goal, hill climbing must have decided
```

Comment:
Nit: This should use a simple `//` comments.

Reply:
I had used the `///` so that I can use `<see cref="..."/>` style reference for easier navigation and renames.